src/main/protobuf/failure_details.proto

```diff
@@ -1195,6 +1196,7 @@ message Analysis {
     CONFIGURED_VALUE_CREATION_FAILED = 18 [(metadata) = { exit_code: 1 }];
     INCOMPATIBLE_TARGET_REQUESTED = 19 [(metadata) = { exit_code: 1 }];
     ANALYSIS_FAILURE_PROPAGATION_FAILED = 20 [(metadata) = { exit_code: 1 }];
+    ANALYSIS_CACHE_DISCARDED = 21 [(metadata) = { exit_code: 49 }];
```

Comment:
What's the context for this field in the current snapshot?

Reply:
I'm sorry, I incorrectly misinterpreted one of the diffs I reviewed. Don't make up a new exit code -- Please switch this back to 1 - sorry for the churn here.